src/game_constants.h

```diff
@@ -55,6 +55,9 @@ static constexpr int OMAPY = OMAPX;
 // Size of a square unit of terrain saved to a directory.
 static constexpr int SEG_SIZE = 32;
 
+// Size of a square unit of tile memory saved in a single file, in mm_submaps.
+#define MM_REG_SIZE 8
```

Comment:
```suggestion
static constexpr int MM_REG_SIZE = 8;
```
It saddens me to introduce yet another coordinate system, but I guess it's not the end of the world.